### DISCUSSION TITLE
Utilize UIMenu popups on iOS in some spots

### DISCUSSION THREAD

**Discussion by hawkrives** [StoDevX/AAO-React-Native#5577] [Ideas]
See https://github.com/dominicstop/react-native-ios-context-menu

**Comment by hawkrives** [comment]
We may need to maintain our own fork, as this one seems to have stalled ~10 months ago.

**Comment by drewvolz** [comment]
Until we're ready to tackle this, would it make sense to move this to discussions and out of issues?

**Comment by hawkrives** [comment]
Sure, that sounds good! (GH mobile doesn't have that button yet. Do you mind switching it?)

**Comment by drewvolz** [comment]
iOS dependency was added to the repo in #5955.
